### PR TITLE
Adds the S-3B Viking

### PR DESCRIPTION
**Pull request by awinterquest**
Adds support for the non-tanker version of the S-3B Viking from pydcs

### CODE REVIEW

File: gen/flights/ai_flight_planner_db.py [hunk 1]
```diff
@@ -209,6 +210,7 @@
     WingLoong_I,
     MQ_9_Reaper,
     RQ_1A_Predator,
+    S_3B,
```
Comment (DanAlbert) [inline gen/flights/ai_flight_planner_db.py:213]: This list is in priority order. I don't think the S-3 is the worst choice for CAS. Not sure what loadout capabilities it has, but it's probably a better choice than most of the helicopters?

File: gen/flights/ai_flight_planner_db.py [hunk 2]
```diff
@@ -312,6 +315,7 @@
     AV8BNA,
     Ju_88A4,
     C_101CC,
+    S_3B,
```
Comment (DanAlbert) [inline gen/flights/ai_flight_planner_db.py:318]: Same here.

**Review by DanAlbert — CHANGES_REQUESTED** [review]
Cool, hadn't realized we we're missing this. Needs a couple changes to priorities but other than that looks good.

File: gen/flights/ai_flight_planner_db.py (hunk 1 above)
Comment (awinterquest) [reply gen/flights/ai_flight_planner_db.py:213]: It is indeed, didn't know about the priority order, I'll check for a better spot

**Pull request closed without merging**